mobile/src/translations/en.json

```diff
@@ -65,7 +65,8 @@
       "create": "Create",
       "next": "Next",
       "ok": "OK",
-      "save": "Save"
+      "save": "Save",
```

Comment:
We could use i18n for this here, too

Reply:
But it is tranlations for i18n, what's wrong?

Reply:
My fault. I mean i18n Manager
https://electronjs.org/apps/i18n-manager

Reply:
Maybe it's overhead? We have a task, that check translations, sooooo

Reply:
Maybe it's overhead? We have a task, that check translations, sooooo

Reply:
Yeah, maybe on mobile it's already inbuilt.  But for the ng wallet that manager is quite nice. We have almost 1000 items in 30 or so languages. That manager does auto translation using Google Translate.  Quite efficient